SetupScript.sh

```diff
@@ -29,8 +29,8 @@ fi
 
 #begin QT install
 apt-get update
-yes | apt-get upgrade
-yes | apt-get install libfontconfig1-dev libdbus-1-dev libfreetype6-dev libudev-dev libicu-dev libsqlite3-dev
+-y apt-get upgrade
+-y apt-get install libfontconfig1-dev libdbus-1-dev libfreetype6-dev libudev-dev libicu-dev libsqlite3-dev
```

Comment:
-y is actually a flag of apt-get, so thebcommand would be apt-get --yes --force-yes

--force-yes is there to get rid of some other prompt about keyrings.